marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/FlowController.java

```diff
@@ -187,6 +188,31 @@ private RunFlowResponse runStepsInAFlow(String flowName, MultipartFile[] uploade
         return runFlowResponse;
     }
 
+    private String[] getStepNumbers(String flowName, String... stepNames) {
+        if(stepNames == null || stepNames.length == 0) {
```

Comment:
Here you can use ArrayUtils.isEmpty (from apache library)

Reply:
@aniamy I think it is much more readable and easy to handle this way instead of using a third-party library. We rarely use this in our codebase as well.